frontend/src/components/AdminPanel/EditCatalogs/CategoryList.tsx

```diff
@@ -24,7 +25,14 @@ const CategoryList = ({
     const [showDeleteCategoryDialog, setShowDeleteCategoryDialog] =
         useState<boolean>(false);
     const [categoryToDelete, setCategoryToDelete] = useState<any>();
-    const deleteCategory = (category: any) => {
+    const [categoryToDeleteContainsQuestions, setCategoryToDeleteContainsQuestions] = useState<boolean>();
+
+    const deleteCategory = async (category: any) => {
+        const categoryContainsQuestions = await listQuestionsByCategoryID(category.id)
+        .then((response: any) => {
+            return response.result.length > 0
+        })
```

Comment:
Her antar jeg at responsen er av typen `Response`, er det nødvendig å ta høyde for at responsen kan være `Failure`? Isåfall, hvordan?

Ref `export type ApiResponse<T> = Response<T> | Failure;` i [adminApi.tsx](https://github.com/knowit/kompetansekartlegging-app/blob/main/frontend/src/components/AdminPanel/adminApi.tsx)

Reply:
Jeg tror det skal gå fint uten å håndtere Failure